### ISSUE TITLE
Dependency on com.github.jitpack:gradle-simple:1.0

### ISSUE THREAD

**Issue by theovier** [rockbite/talos#78]
Hi there,

I am using Maven and I noticed that the talos dependency itself

```         
<dependency>
     <groupId>com.talosvfx</groupId>
      <artifactId>talos-libgdx</artifactId>
      <version>1.4.0</version>
</dependency>
``` 
has a dependency on `com.github.jitpack:gradle-simple:1.0`. 

![image](https://user-images.githubusercontent.com/13664498/157939028-e6de7d74-6595-46c7-95af-99264319ef43.png)

To me that seems strange as it seems to be an [example gradle project](https://github.com/jitpack/gradle-simple).

I have not looked at the source code yet, but can someone comment on whether this dependency is in fact needed?

Thanks!

**Comment by Tom-Ski** [comment]
Yep it shouldn't be included.

**Comment by Tom-Ski** [comment]
Closed via #80
